src/controllers/movie.js

```diff
@@ -0,0 +1,72 @@
+const { Prisma } = require("@prisma/client");
+const prisma = require("../utils/prisma");
+
+const getAllMovies = async (req, res) => {
+  const movies = await prisma.movie.findMany ({
+    include: {
+        screenings: true,
+      },
+     });
+
+  return res.status(200).json({ movies });
+};
+
+const createMovie = async (req, res) => {
+  const { title, runtimeMins } = req.body;
+
+  const movie = await prisma.movie.create({
+    data: {
+      title: 'Top Gun',
+      runtimeMins:110,
```

Comment:
Here you should have title & runtimeMins or alternatively title: title & runtimeMins: runtimeMins.